ui/app/AppLayouts/Chat/views/ChatMessagesView.qml

```diff
@@ -222,41 +220,24 @@ Item {
             visible: chatLogView.visibleArea.heightRatio < 1
         }
 
-        Button {
-            id: scrollDownButton
-
+        ChatAnchorButtonsPanel {
             anchors.bottom: parent.bottom
             anchors.right: parent.right
             anchors.rightMargin: Style.current.padding
 
-            visible: false
-            height: 32
-            width: arrowImage.width + 2 * Style.current.halfPadding
-
-            background: Rectangle {
-                color: Style.current.buttonSecondaryColor
-                border.width: 0
-                radius: 16
+            mentionsCount: d.chatDetails ? d.chatDetails.notificationCount : 0
+            recentMessagesButtonVisible: {
+                chatLogView.contentY // trigger binding on contentY change
+                return chatLogView.contentHeight - (d.scrollY + chatLogView.height) > 400
```

Comment:
what is 400?

Reply:
I didn't introduce it. This piece of code was just moved.